worldloader.js

```diff
@@ -58,7 +58,7 @@ ZZTWorldLoader.prototype.parseWorldData = function(stream)
    /* board information then starts at offset 512 */
    stream.position = 512;
 
-   for (var i = 0; i < world.numBoards; ++i)
+   for (var i = 0; i <= world.numBoards; ++i)
```

Comment:
Deals with one-off bug.  The number of boards reported is one less since the title screen isn't included in the number.